dev/com.ibm.ws.microprofile.openapi_fat/fat/src/com/ibm/ws/microprofile/openapi/fat/OpenApiActions.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.openapi.fat;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.FeatureSet;
+import componenttest.rules.repeater.FeatureSet.FeatureSetBuilder;
+import componenttest.rules.repeater.JakartaEE9Action;
+import componenttest.rules.repeater.MicroProfileActions;
+import componenttest.rules.repeater.RepeatActions;
+import componenttest.rules.repeater.RepeatTests;
+
+public class OpenApiActions {
+
+    public static final String MP_OPENAPI_31_ID = JakartaEE9Action.ID + "_MpOpenApi_31";
```

Comment:
Wouldn't it be more accurate for this to be?
```suggestion
    public static final String MP_OPENAPI_31_ID = MicroProfileActions.MP50_ID + "_MpOpenApi_31";
```

Reply:
Yeah ok, I think that would make sense

Reply:
Updated